challenge.js

```diff
@@ -42,7 +44,7 @@
  */
 
 // ANSWER
-
+document.getElementById('E')
```

Comment:
Nope, we're looking for a CSS Selector here. One possible answer might be `body > div > p`.